.gitignore

```diff
@@ -51,3 +51,10 @@ android/keystores/debug.keystore
 
 # Build
 dist/
+
+# TypeScript-generated files
+*.d.ts
+!src/react-navigation.d.ts
```

Comment:
Maybe we should put it inside `types/` folder instead of putting in `src/`? It's bit weird to have generated files inside of `src/`.

Also does `npm publish` upload them if you add to `.gitignore`?

Reply:
as far as I understand, yes. `npm publish` will ignore files listed in `.gitignore`.
That being said, I guess we can also use the `.npmignore` file to "negate" that?

some reference: https://stackoverflow.com/a/47127547/8595941

Reply:
oh right!, i added that tho. sorry i was mixing ideas :(

Reply:
hmm I don't see it?